web-share/share-null-manual.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare Test: Share null and undefined values</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="resources/manual-helper.js"></script>
+  </head>
+  <body>
+    <script>
+        setup({explicit_timeout: true});
+
+        // Expect null to be converted into the string 'null'. (This isn't good
+        // practice, but it's how Web IDL specifies conversion.) On the other
```

Comment:
The note isn't saying it's bad practice for the spec to be defined this way, it's saying it's bad practice for a developer to rely on this (but that it's correct). Regardless, dropped this comment.

> If you want null to be handled properly, why not just do so in the IDL? i.e.:

This was a deliberate change in response to feedback: [#19](https://github.com/WICG/web-share/issues/19). I explicitly want `null` to be converted to the string `"null"`, just like any other object value is converted into a string. This test just tests that edge case.

I initially declared these nullable but since I've learned more about WebIDL, I think it makes sense to not allow `null` here. We already allow omission / `undefined`. Allowing `null` as a legal value is just an additional case that there is no need for.